taskcluster/app_services_taskgraph/transforms/module_build.py

```diff
@@ -14,17 +17,16 @@ def build_task(config, tasks):
     for task in tasks:
         module_info = task["attributes"]["buildconfig"]
         name = module_info["name"]
-        version = module_info["version"]
+        version = get_version()
```

Comment:
Note: Could've kept the version in the `attributes` but we're going to need a separate function for beetmover anyway so I thought it's cleaner to separate it like this.